conftest/helm-conftest.yaml

```diff
@@ -2,7 +2,18 @@ apiVersion: tekton.dev/v1beta1
 kind: Task
 metadata:
   name: helm-conftest
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tags: test
+    displayName: "helm conftest"
```

Comment:
when did we decide on these tag and displayName annotations? i think we should discuss before adding and requiring these

Reply:
if we do go ahead, would it make sense to preface these with tekton.dev as well?

Reply:
How about we address those changes in a seperate PR ?

Reply:
@PuneetPunamiya agree but lets create an issue before the PR is merged so that we won't miss addressing those or at least bring this up for discussion in WG meeting?